app/test_main.py

```diff
@@ -1,3 +1,22 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        (-1, -1, [0, 0]),
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (23, 23, [1, 1]),
+        (100, 100, [21, 17]),
+    ]
+)
```

Comment:
you should add more data for tests to check all age boundaries. Also you should add descriptive ids